packages/wallet/test/KeyManagement/InMemoryKeyManager.test.ts

```diff
@@ -1,28 +1,26 @@
-import { CSL, Cardano } from '@cardano-sdk/core';
+import { Cardano } from '@cardano-sdk/core';
```

Comment:
Looks like you have this noted given the temporary export as the old name, but just a reminder to rename to `InMemoryKeyAgent.test.ts`